roles/pulp_common/tasks/repos.yml

```diff
@@ -134,7 +134,8 @@
     name: "{{ __pulp_pkg_repo_name }}"
     description: "{{ __pulp_pkg_repo_name }}"
     baseurl: "{{ pulp_pkg_repo }}"
-    gpgcheck: false
+    gpgcheck: true
+    gpgkey: "{{ pulp_pkg_repo }}/../../GPG-RPM-KEY-pulpcore"
```

Comment:
This is specific for theforeman.org repos. So long as they are the only ones that exist, I am OK with this. Let's not over-engineer.

Reply:
I made it slightly more configurable, also because I'll need to be able to turn of gpgcheck for my pipeline tests in pulplift.